tests/test_complete.py

```diff
@@ -140,7 +140,10 @@ def test_complete(self):
         # Verify that we got the expected completions back.
         self.assertIsNotNone(completions)
         expected = ['a\tint a', 'a']
-        self.assertIn(expected, completions)
+        if isinstance(completer, CompleterBin):
```

Comment:
I guess it would be cool to have more tests on this. At least, that destructors are never there.